tests/functional/spec/save-come-back.spec.js

```diff
@@ -0,0 +1,118 @@
+import reportingPeriod from '../pages/surveys/rsi/0102/reporting-period.page'
+import retailTurnoverPage from '../pages/surveys/rsi/0102/retail-turnover.page'
+import internetSalesPage from '../pages/surveys/rsi/0102/internet-sales.page'
+import changeInRetailTurnover from '../pages/surveys/rsi/0102/changes-in-retail-turnover.page'
+import SummaryPage from '../pages/summary.page'
+
+const expect = chai.expect
+
+describe('RSI - Save and restore test', function() {
+
+  it('Given I am completing survey 0102, when I select save and come back later, then I am redirected to sign out page, showing a message that my survey has been saved and I cant continue without re-authenticating', function() {
+
+    // Given I am completing survey 0102
+    const collectionId = getRandomString(5)
+    startQuestionnaire('1_0102.json', 'test_user', collectionId)
+    const url = browser.getUrl();
+    reportingPeriod.setFromReportingPeriodDay('01')
+      .setFromReportingPeriodMonth(5)
+      .setFromReportingPeriodYear('2016')
+      .setToReportingPeriodDay('01')
+      .setToReportingPeriodMonth(5)
+      .setToReportingPeriodYear('2017')
+      .submit()
+
+    // When I select save and come back later
+    retailTurnoverPage.saveComeBack()
+
+    // Then I am redirected to sign out page, showing a message that my survey has been saved and I cant continue without re-authenticating
+    expect(signOut.isOpen(), 'sign out page should be open').to.be.true
+    browser.url(url)
+    expect(browser.getSource()).to.contain('Error 401')
+    })
+
+    it('Given I am completing survey 0102 and have outstanding validation failures, when I select save and come back later, then I must correct my response before my questionnaire can be saved and I am signed out', function() {
+
+    // Given I am completing the EQ and have outstanding validation failures
+    const collectionId = getRandomString(5)
+    startQuestionnaire('1_0102.json', 'test_user', collectionId)
+    reportingPeriod.setFromReportingPeriodDay('01')
+      .setFromReportingPeriodMonth(5)
+      .setFromReportingPeriodYear('2016')
+      .setToReportingPeriodDay('01')
+      .setToReportingPeriodMonth(5)
+      .setToReportingPeriodYear('2017')
+      .submit()
+    retailTurnoverPage.setRetailTurnover('test')
+
+    // When I select save and compe back later
+    retailTurnoverPage.saveComeBack()
+
+    // Then I must correct my response before my questionnaire can be saved and I am signed out
+    expect(retailTurnoverPage.getAlertText()).to.contain('Please only enter whole numbers into the field.')
+
+    retailTurnoverPage.setRetailTurnover('2000')
+      .saveComeBack()
+
+    expect(signOut.isOpen(), 'Sign out page should be open').to.be.true
+    })
+
+  it('Given I am completing completing survey 0102 and I have not completed mandatory questions, when I select save and come back later, then the missing mandatory questions will be ignored and I will be redirected to the exit page', function() {
```

Comment:
The household question has a bespoke view, and therefore its own call to save and come back, we will need both to cover it